framework/decode/vulkan_rebind_allocator.cpp

```diff
@@ -853,17 +855,20 @@ VkResult VulkanRebindAllocator::AllocateAHBMemory(MemoryAllocInfo* memory_alloc_
     dedicatedAllocateInfo.pNext = &importAHBInfo;
     dedicatedAllocateInfo.image = image;
 
-    VkMemoryRequirements memoryRequirements;
-    functions_.get_image_memory_requirements(device_, image, &memoryRequirements);
+    VkAndroidHardwareBufferPropertiesANDROID androidHardwareBufferProperties;
+    androidHardwareBufferProperties.sType = VK_STRUCTURE_TYPE_ANDROID_HARDWARE_BUFFER_PROPERTIES_ANDROID;
+    androidHardwareBufferProperties.pNext = nullptr;
+    functions_.get_android_hardware_buffer_properties(
```

Comment:
Should we assert if `function_.get_android_hardware_buffer_properties` is `nullptr`?  Otherwise, looks good to me.